dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerTest.kt

```diff
@@ -60,13 +60,13 @@ class CompanyDataControllerTest {
     }
 
     @Test
-    @Suppress("kotlin:S138")
-    fun `post two dummy companies and check if the distinct endpoint returns all values`() {
+    @Suppress("kotlin:S138") // TODO who included this and why? is it required?
```

Comment:
Question to everyone?

Reply:
Das unterdrückt den Fehler "Function too long". Aber wenn ich das richtig sehe ist der Test jetzt wieder kürzer als 20 Zeilen. Kann also weg.